docs/architecture.md

```diff
@@ -0,0 +1,63 @@
+# Architecture
+
+- Raw data comes in whatever form
```

Comment:
Re: format data comes in on, how close to reality do we want to hew here, and how aspirational do we want to be? Reality _could_ be we take in anything from which we can get counts of taxa per location/group per day/timestep. Reality is that we read in line-list data directly from one specific source.